Write Your Own Crontab Tool/Sercice/ValidationService.swift

```diff
@@ -53,18 +53,18 @@ private extension ValidationService {
     
     func validateStringValue(_ value: String, field: Field) -> CustomError? {
         var error: CustomError?
-        
-        if value.count == 1 {
+        let separators = value.filter({ field.separationCharacters.contains(String($0)) })
```

Comment:
This can be better
```
if value.count == 1 && !field.separationCharacters.contains(value) {
            let singleSpecialCharacters = field.singleSpecialCharacters
            if !singleSpecialCharacters.contains(value) {
                error = CustomErrorService.shared.generateError(for: field, value: value)
            }
        } else {
            let separators = value.filter({ field.separationCharacters.contains(String($0)) })
            if !separators.isEmpty {
                error = validateListSeparator(value, field: field) ??
                validateRangeSeparator(value, field: field) ??
                validateStepSeparator(value, field: field) ??
                validateHashtagSeparator(value, field: field)
            } else {
                error = validateSymbolValue(value, field: field)
            }
        }
```